OWML.ModHelper.Menus/ModsMenu.cs

```diff
@@ -54,6 +54,10 @@ public void Initialize(IModMainMenu mainMenu)
             var modsMenu = CreateModsMenu(optionsMenu);
             modsButton.OnClick += () => modsMenu.Open();
             Menu = mainMenu.Menu;
+
+            InitConfigMenu(_menus.OwmlMenu, optionsMenu);
+            var owmlButton = modsButton.Duplicate("OWML");
```

Comment:
Maybe call it something else, like "Mod Loader" or "Mod Loader Settings"?

Reply:
I would say "Settings" is excessive. naming it "Mod Loader" sounds like a good idea though

Reply:
Just tested with "Mod Loader". I tried thinking as a player, and when I saw it I expected it to be something else, maybe a way to load mods. So I reverted it to "OWML" again... 🤷‍♂️